_pages/dev/features/token-revocation.md

```diff
@@ -14,37 +14,33 @@ feature:
+When a user signs out of the storefront, Spartacus requests the revocation of the user authentication token so that the OCC session is terminated. This is a security improvement.
 
-Here is what to expect:
+Token revocation works as follows:
 
 - When a customer signs out of the storefront, a token revocation request is sent.
 - When an ASM customer support agent signs out, a token revocation request is sent.
-- When a customer emulated with ASM signs out, a token revocation is _not_ sent. The customer support agent's token is used in emulation sessions and we only want to revoke it when the agent himself signs out.
+- When a customer who is emulated with ASM signs out, a token revocation is _not_ sent. The customer support agent's token is used in emulation sessions, so Spartacus only revokes the token when the agent signs out.
 
-If the token revocation request fails, Spartacus will fail silently since there is no action it can make to recover. For example requesting the revocation of an already expired token will result in a http 401 response. Even if the server sends an error response, Spartacus will fail silently.
+If the token revocation request fails, Spartacus fails silently, because there is no action it can make to recover. For example, requesting the revocation of an already-expired token results in an HTTP 401 response. Even if the server sends an error response, Spartacus will fail silently.
 
 ## Requirements
 
-### Spartacus
+For the front end, you need version 1.4 or newer of the Spartacus libraries.
 
-You need Spartacus v1.4.0
+For the back end, token revocation is available with version 1905.6 or newer of SAP Commerce Cloud.
 
-### Backend
+Spartacus 1.4 can still be used with a SAP Commerce Cloud back end that is older than 1905.6, but in the front end, token revocation will fail silently if the back end does not support it.
 
-Token revocation is available on backends with version 1905.6 and above.
-
-Spartacus 1.4.0+ can still be used with backends with a version lower than 1905.6. In Spartacus, token revocation will fail silently if the backend does not support it.
-
-Under the hood, a backend that does not support token revocation will return an http 302 redirect response on a token revocation request. The backend's behaviour on a request to a non-existing authorization-related endpoint is to return a 302 redirect code pointing to the `login.jsp` page.
+Under the hood, a back end that does not support token revocation returns an HTTP 302 redirect response if it receives a token revocation request. If a request is sent to an authorization-related endpoint that does not exist, the back end returns a 302 redirect code that points to the `login.jsp` page.
 
 ## Enabling Token Revocation
 
-Token revocation in Spartacus is always enabled in v1.4.0+.
+Token revocation is enabled by default in Spartacus version 1.4 or newer.
```

Comment:
`by default` implies that it can be turned off.  Token revocation can't be turned off.   That was what I was trying to communicate with "always enabled".  There is probably a way to communicate it that is less ambiguous.

Reply:
Text has been updated as follows:

> Token revocation is automatically enabled in Spartacus version 1.4 or newer. It cannot be disabled.